firmware/new_serial.py

```diff
@@ -7,7 +7,7 @@
 """
 
 try:
-    ser = int(file(".serial", 'rb').read(), 16) #+ 1
+    ser = int(file(".serial", 'rb').read(), 16) + 1
```

Comment:
this is currently commented out so that the serial number only increases when flashing the bootloader, no longer every time we build new firmware.  it's ugly and in limbo, but we'll fix it correctly when the firmware numbers are as they should be. ;)

Reply:
sorry, this needs to go as well.